src/components/ProjectSection/ProjectData.ts

```diff
@@ -41,8 +43,10 @@ export const PastProjects: Project[] = [
     website: website,
     image: bandist,
     pmName: "Nurgul Akhshatayeva",
+    year: "",
```

Comment:
It would be valuable to include the years for all the projects (we may display them different based on year in the future)